nrf_modem/README.rst

```diff
@@ -2,33 +2,29 @@
 
 Modem library
 #############
-The Modem library is the primary interface for operating the nRF9160 modem, establishing the LTE-M and NB-IoT connections, and receiving the position data (GPS).
-The library abstracts away the details of memory management and synchronization, allowing you to focus on writing the application using standardized API calls.
 
-The library aims to be compatible with the socket standard to simplify writing and porting of existing applications.
-The library supports up to eight networking sockets.
+The Modem library is the primary interface for operating the nRF9160 modem, establishing the LTE-M and NB-IoT connections, and receiving the position data (GPS).
+The library aims to be compatible with the POSIX socket APIs to simplify writing and porting of existing networking applications.
 
-To get started with socket programming, you should check out some introductions and tutorials, such as `Beej's Guide to Network Programming`_.
+To get started with socket programming, you must check out some introductions and tutorials, such as `Beej's Guide to Network Programming`_.
 
-The Modem library has been designed in such way that you can port it to any RTOS.
-Therefore, procedures that would normally require some kind of OS interaction, like going into sleep mode and IRQ reprioritization, have been moved out of the library to a :file:`.c` file in which OS-specific integration can be implemented.
+The Modem library is portable, you can be use it with any RTOS.
```

Comment:
`portable, you can use it with any RTOS.` or `portable, it can be used with any RTOS.`